easybuild/easyconfigs/j/Java/Java-1.8.eb

```diff
@@ -0,0 +1,23 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'Bundle'
+
+name = 'Java'
+version = '1.8'
```

Comment:
Hmm, would `8` make more sense here? Whatever the source tarball may be named, the actual version is actually `Java` 8, right?

Reply:
I don't really care....
[j-perdue@tlogin-0502 (08:50) ~]$ ml Java/1.8.0
[j-perdue@tlogin-0502 (08:50) ~]$ java -version
java version "1.8.0_172"
Java(TM) SE Runtime Environment (build 1.8.0_172-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.172-b11, mixed mode)

I just went with the naming standard that EB already used. 1.8 seems to make some sense.

Reply:
might be a good idea for 9

[j-perdue@login2 (08:59) ~]$ ml Java/9.0
[j-perdue@login2 (09:00) ~]$ java -version
java version "9.0.4"
Java(TM) SE Runtime Environment (build 9.0.4+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.4+11, mixed mode)

Reply:
This thought came up mostly due to the new Java 10, jumping from 1.8 to 10 would be sort of strange.

Also, I think using `('Java', '8')` would make it more clear that we're using a wrapper, since you'd expect a `major.minor`...

Reply:
At www.oracle.com/technetwork/java/javase/downloads they refer to it as `Java SE 8` and `Java 8u181`.

What `java -version` prints is probably mostly a concern with backwards compatibility, which they broke in Java 9.x

Reply:
your call... I don't care either way